pkgs/tools/misc/nbqa/default.nix

```diff
@@ -0,0 +1,93 @@
+{ black
+, blacken-docs
+, fetchFromGitHub
+, lib
+, python3
+, ruff
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "nbqa";
+  version = "1.7.0";
+
+  src = fetchFromGitHub {
+    owner = "nbQA-dev";
+    repo = "nbQA";
+    rev = version;
+    hash = "sha256-CTF5HisBS44Ta18cVT04UrMIF30WmEBwZBGW7fkKXwk=";
+  };
+
+  passthru.optional-dependencies = {
+    # Based on https://github.com/nbQA-dev/nbQA/blob/1.7.0/setup.cfg#L47-L56
```

Comment:
```suggestion
```
This is not a meaningful comment because it is obvious.

Reply:
I don't think it's obvious, but I've removed it.